jest.config.js

```diff
@@ -14,5 +14,8 @@ module.exports = {
     "ts-jest": {
       "tsconfig": "jest.tsconfig.json"
     }
+  },
+  "moduleNameMapper":{
```

Comment:
Can you explain to me the rationale behind what we're doing here?

Reply:
This maps modules with extensions css|less|sass|scss to the mock module (which in turn exports nothing) - thereby decoupling the tests from the sass compilation and styles. I guess alternatively one could get Jest to compile the scss but I thought this would be easier?